Prox/Prox/PlaceDetails/PlaceDetailsCardViewController.swift

```diff
@@ -187,6 +195,8 @@ class PlaceDetailsCardViewController: UIViewController {
             let url = URL(string: urlString) else { return }
         if !OpenInHelper.open(url: url) {
             print("lol unable to open web address")
+        } else {
+            Analytics.logEvent(event: AnalyticsEvent.EVENT_BANNER_LINK, params: [:])
```

Comment:
This will only work if there is a URL attached to the event. Is there value in logging that people attempted to interact with an event that had no URL?